examples/backups/vmi/07-restore-to-copy.sh

```
@@ -0,0 +1,68 @@
+  namespace: ${NAMESPACE}
+spec:
+  backupRef:
+    name: test-backup
+  targetApplicationRef:
+    apiGroup: apps.cozystack.io
+    kind: VMInstance
+    name: test
+  options: # runtime.RawExtension, typed based on targetApplicationRef and current controller implementation (for additional restore options)
+    targetNamespace: ${TARGET_NAMESPACE} # when set to a different namespace, triggers cross-namespace restore via Velero namespaceMapping
+    failIfTargetExists: true # if true, restore will fail when the target resource already exists
+    keepOriginalPVC: false # renames original VMI PVC before restore to `<name>-orig-<hash>`, only for in-place restore
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)